### PR TITLE
Semantic ui theming

### PR DESCRIPTION
**Pull request by RoseannaM**
This PR introduces Semantic UI's theming build process. We can now modify the LESS variables and style overrides used by Semantic UI to generate CSS that we use instead of Semantic UI's default CSS. This lets us change the components' default styling.

### CODE REVIEW

**Comment by bcomnes** [comment]
Will review Monday. 🏖

File: .gitignore [hunk 1]
```diff
@@ -7,10 +7,13 @@ node_modules
 coverage
 
 # production
-build
+/build
```
Comment (bcomnes) [inline .gitignore:10]: I forget, how are absolute looking paths handled in gitignore again?

File: package.json [hunk 2]
```diff
@@ -21,12 +22,13 @@
...
     "semantic-ui-react": "^0.62.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build:css": "cd src/semantic && gulp build",
+    "start": "yarn build:css && react-scripts start",
+    "build": "yarn build:css && react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "lint": "npm run lint:js",
```
Comment (bcomnes) [inline package.json]: Lets pick yarn or npm, not both (whichever we are using on these).

**Review by bcomnes — CHANGES_REQUESTED** [review]
Look good. Semantic UI is huge, so I didn't look through every file added here.  If you have specific design decisions you want feedback on please let me know.

File: package.json (hunk 2 above)
Comment (bcomnes) [reply package.json]: Looks like we are going with yarn, so once we make that consistent, good to go.

File: .gitignore (hunk 1 above)
Comment (RoseannaM) [reply .gitignore:10]: There's a folder called "build" inside the generated semantic ui content that we don't want to be ignored, so I changed this to an absolute path so we would only ignore this build folder at the top level.

Comment (bcomnes) [reply .gitignore:10]: 👌

**Review by bcomnes — APPROVED** [review]

**Comment by bcomnes** [comment]
LGTM!

**Pull request merged**